### PR TITLE
Update native modules

### PR DESCRIPTION
**Pull request by eternoendless**
Updated modules:

* ps_faviconnotificationbo: v1.0 -> v2.0
* ps_linklist: v2.1.6 -> 3.0.3
* ps_themecusto: v1.0.9 -> v1.0.10
* welcome: v5.1.0 -> v6.0

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Updated native modules to latest
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | n/a
| How to test?  | Make sure you are able to install a clean shop

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13735)

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -3948,16 +3957,16 @@
         },
         {
             "name": "prestashop/ps_themecusto",
-            "version": "v1.0.9",
+            "version": "v1.0.10",
```
Comment (Quetzacoalt91) [inline composer.lock:3960]: Is this expected?

Comment (jolelievre) [reply composer.lock:3960]: If there was an update I don't think it's a problem to integrate it in the zip

File: composer.json [hunk 2]
```diff
@@ -58,11 +58,11 @@
         "prestashop/ps_customtext": "^4",
         "prestashop/ps_emailsubscription": "^2",
         "prestashop/ps_facetedsearch": "^2",
-        "prestashop/ps_faviconnotificationbo": "^1",
+        "prestashop/ps_faviconnotificationbo": "^2",
         "prestashop/ps_featuredproducts": "^2",
         "prestashop/ps_imageslider": "^3",
         "prestashop/ps_languageselector": "^2",
-        "prestashop/ps_linklist": "^2",
+        "prestashop/ps_linklist": "^3",
```
Comment (Quetzacoalt91) [inline composer.json:65]: Looks like we cannot accept that : 

```
Warning: Uncaught Symfony\Component\Debug\Exception\ContextErrorException: Warning: require_once(/home/travis/build/PrestaShop/PrestaShop/modules/ps_linklist/vendor/autoload.php): failed to open stream: No such file or directory in /home/travis/build/PrestaShop/PrestaShop/modules/ps_linklist/ps_linklist.php:30
```

https://travis-ci.org/PrestaShop/PrestaShop/jobs/530335151#L632

Comment (eternoendless) [reply composer.json:65]: It should work now 🤞

**Review by Quetzacoalt91 — APPROVED** [review]

**Pull request merged**